test/unit/Cover/buyCover.js

```diff
@@ -712,4 +711,372 @@ describe('buyCover', function () {
       ),
     ).to.be.revertedWith('Cover: The selected pools ran out of capacity');
   });
+
+  it('reverts if system is paused', async function () {
+    const { cover, master } = this;
+
+    const {
+      governanceContracts: [gv1],
+      members: [coverBuyer, stakingPoolManager],
+    } = this.accounts;
+
+    const productId = 0;
+    const coverAsset = 0; // ETH
+    const period = 3600 * 24 * 364; // 30 days
+
+    const amount = parseEther('1000');
+    const targetPriceRatio = '260';
```

Comment:
just checked and this pool fixture and the createStakingPool call ar all the same for all the tests currently.

It makes sense to have  a `beforeEach` section which is just creating this default pool to avoid this code duplication.

We will add tests for buying across multiple pools. in those tests we can add the extra pool there but we always need at least this 1 pool.

initializing all these values in a fixture to be used in the rest of the test code would strip this section a code from each test.

Reply:
So let's do a beforeEach with all this code including the `createStakingPool` 

create a fixture for the values for the pool so that values are not initialized in every test

Reply:
Thanks for the suggestion! I refactored the code, now the tests code is cleaner